res/values/strings.xml

```diff
@@ -1677,46 +1664,46 @@
     <string name="conversation_activity__enable_signal_messages">ENABLE SIGNAL MESSAGES</string>
     <string name="SQLCipherMigrationHelper_migrating_signal_database">Migrating Signal database</string>
     <string name="PushDecryptJob_new_locked_message">New locked message</string>
     <string name="PushDecryptJob_unlock_to_view_pending_messages">Unlock to view pending messages</string>
     <string name="ExperienceUpgradeActivity_unlock_to_complete_update">Unlock to complete update</string>
     <string name="ExperienceUpgradeActivity_please_unlock_signal_to_complete_update">Please unlock Signal to complete update</string>
     <string name="enter_backup_passphrase_dialog__backup_passphrase">Backup passphrase</string>
-    <string name="backup_enable_dialog__backups_will_be_saved_to_external_storage_and_encrypted_with_the_passphrase_below_you_must_have_this_passphrase_in_order_to_restore_a_backup">Backups will be saved to external storage and encrypted with the passphrase below. You must have this passphrase in order to restore a backup.</string>
-    <string name="backup_enable_dialog__i_have_written_down_this_passphrase">I have written down this passphrase. Without it, I will be unable to restore a backup.</string>
-    <string name="registration_activity__restore_backup">Restore backup</string>
+    <string name="backup_enable_dialog__backups_will_be_saved_to_external_storage_and_encrypted_with_the_passphrase_below_you_must_have_this_passphrase_in_order_to_restore_a_backup">Backup files will be saved to internal storage and will be encrypted with the passphrase below. You must have this passphrase in order to restore a backup.</string>
+    <string name="backup_enable_dialog__i_have_written_down_this_passphrase">I have written down this passphrase. Without it, I will be unable to restore from a backup.</string>
+    <string name="registration_activity__restore_backup">Restore from backup</string>
     <string name="registration_activity__skip">Skip</string>
     <string name="registration_activity__register">Register</string>
-    <string name="preferences_chats__chat_backups">Chat backups</string>
-    <string name="preferences_chats__backup_chats_to_external_storage">Backup chats to external storage</string>
-    <string name="preferences_chats__create_backup">Create backup</string>
+    <string name="preferences_chats__chat_backups">Let Signal create backups</string>
+    <string name="preferences_chats__backup_chats_to_external_storage">Allow Signal to create encrypted backup files of your chats and media to /Signal/Backups on your phone\'s internal storage. If enabled Signal will create backups automatically every 24 hours. Only the two most recent successful backups will be kept.</string>
```

Comment:
Two suggestions here:
1. Comma after "If enabled"
1. Replace "automatically every 24 hours" by "daily"

Reply:
I agree about the comma.

"If enabled, Signal will automatically create daily backups" Confusing.
"If enabled, Signal will automatically create a backup each day" could be done, but unnecessary emphasis to the word "each".
I don't know about this. Do you like it better if it says "will automatically create a backup each day"? Lets not drop the word "automatically"

Reply:
New proposal. "If enabled, Signal will create daily backups automatically"